dev/tests/integration/testsuite/Magento/Email/Model/TemplateTest.php

```diff
@@ -71,18 +71,18 @@ protected function mockModel($filesystem = null)
 
         $this->objectManager->get(\Magento\Framework\App\State::class)->setAreaCode('frontend');
 
-        $this->model->expects($this->any())->method('_getMail')->will($this->returnCallback([$this, 'getMail']));
+        $this->model->expects($this->any())->method('_getMail')->will($this->returnCallback([$this, 'getMessage']));
```

Comment:
Looks like `_getMail()` method doesn't exist anymore. It shouldn't be necessary here then.

Reply:
Thank you @buskamuza for catching this, I overlooked it. Will fix it asap.

Reply:
Hm... probably the test itself still makes some sense. But anything related to `$this->mail` does not.